### PR TITLE
chore(404): no cache for 404

### PR DESCRIPTION
**Pull request by ovflowd**
This PR adds no-cache for 404 pages

### CODE REVIEW

**Comment by richardlau** [comment]
@ovflowd 
```console
# nginx -t
nginx: [emerg] invalid number of arguments in "try_files" directive in /etc/nginx/sites-enabled/00-nodejs.org:39
nginx: configuration file /etc/nginx/nginx.conf test failed
```

**Comment by ovflowd** [comment]
@richardlau done.

File: ansible/www-standalone/resources/config/nodejs.org [hunk 1]
```diff
@@ -34,6 +34,10 @@ server {
         
         try_files $uri $uri/ =404;
     }
+    
+    location = /en/404.html {
+        add_header Cache-Control "no-cache" always;
+    }
```
Comment (targos) [inline ansible/www-standalone/resources/config/nodejs.org:40]: maybe you could have kept the `@name` and used `alias /en/404.html;` ?

Comment (ovflowd) [reply ansible/www-standalone/resources/config/nodejs.org:40]: That would also work.

**Comment by richardlau** [comment]
```console
# nginx -t
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful
```
🎉 

Going to deploy 🤞 .

File: ansible/www-standalone/resources/config/nodejs.org (hunk 1 above)
Comment (ovflowd) [reply ansible/www-standalone/resources/config/nodejs.org:40]: Issue is `alias` would require full path

**Comment by ovflowd** [comment]
@richardlau I've just made an update using `alias` as @targos suggested, not sure which option is better tbh

**Comment by richardlau** [comment]
With the alias 
```console
# nginx -t
nginx: [emerg] the "alias" directive cannot be used inside the named location in /etc/nginx/sites-enabled/00-nodejs.org:39
nginx: configuration file /etc/nginx/nginx.conf test failed
```

**Comment by targos** [comment]
Let's keep it without the alias, then.

**Comment by ovflowd** [comment]
Wait, if the real path is /en/404/index.html ... shouldn't that actually be causing problems? How is it able to find /en/404.html?

File: ansible/www-standalone/resources/config/nodejs.org [hunk 2]
```diff
@@ -35,7 +35,9 @@ server {
         try_files $uri $uri/ =404;
     }
     
-    location = /en/404.html {
+    location @english_404 {
+        alias /home/dist/nodejs/en/404.html;
```
[the remark this answers is not in the document]
Comment (ovflowd) [reply ansible/www-standalone/resources/config/nodejs.org]: Actually, now `/home/www/nodejs/404/index.html`

**Comment by richardlau** [comment]
nginix is still unhappy:
```console
# nginx -t
nginx: [emerg] the "alias" directive cannot be used inside the named location in /etc/nginx/sites-enabled/00-nodejs.org:39
nginx: configuration file /etc/nginx/nginx.conf test failed
```

File: ansible/www-standalone/resources/config/nodejs.org [hunk 3]
```diff
@@ -13,7 +13,7 @@ server {
     keepalive_timeout 60;
     server_tokens off;
     
-    error_page 404 /en/404.html;
+    error_page 404 @english_404;
```
Comment (targos) [inline ansible/www-standalone/resources/config/nodejs.org:16]:
```suggestion
    error_page 404 /404/index.html;
```

Comment (ovflowd) [reply ansible/www-standalone/resources/config/nodejs.org]: Good catch.

**Comment by richardlau** [comment]
```console
# nginx -t
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful
```
🎉

**Review by richardlau — APPROVED** [review]

**Comment by ovflowd** [comment]
I've just merged the PR on `nodejs/nodejs.org`. (@richardlau). Let's wait for the build to finish within the server 👀

**Comment by targos** [comment]
https://github.com/nodejs/build/pull/3226/commits/c7ab83237759f491dac078d9d512060d36ad2f2d is not deployed

**Pull request merged**